src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -35,4 +35,18 @@ CREATE TABLE gene_panel_to_gene
     gene_panel_id LowCardinality(String),
     gene String
 ) ENGINE = MergeTree()
-ORDER BY (gene_panel_id);
\ No newline at end of file
+ORDER BY (gene_panel_id);
+
+CREATE TABLE sample_derived
```

Comment:
this will replace sample_mv?

Reply:
Yea, eventually. We need to finalize all our tables. I was just sticking to our convention